src/tools/clippy/clippy_lints/src/attrs/mod.rs

```diff
@@ -417,12 +417,6 @@ impl_lint_pass!(Attributes => [
     ALLOW_ATTRIBUTES,
     ALLOW_ATTRIBUTES_WITHOUT_REASON,
```

Comment:
NIT, these are now part of the `PostExpansionEarlyAttributes` and are also registered as part of it.

```suggestion
```

Reply:
oops, well fixed that.